packages/insured-bridge-relayer/src/index.ts

```diff
@@ -53,13 +53,21 @@ export async function run(logger: winston.Logger, l1Web3: Web3): Promise<void> {
     const l2Web3 = getWeb3ByChainId(config.activatedChainIds[0]);
     const latestL2BlockNumber = await l2Web3.eth.getBlockNumber();
     const l2StartBlock = Math.max(0, latestL2BlockNumber - config.l2BlockLookback);
-
+    const fallbackL2Web3s = getRetryWeb3sByChainId(config.activatedChainIds[0]);
+    // Note: This will not construct duplicate Web3 objects for URL's in the Retry config that are the same as the
+    // one used to construct l2Web3.
+    logger.debug({
+      at: "AcrossRelayer#index",
+      message: `Constructed ${fallbackL2Web3s.length} fallback L2 web3 providers`,
```

Comment:
I wish you could extract the `rpc` url from a provider. would make the logs here and in the client so much nicer.